src/api_request.py

```diff
@@ -23,7 +23,7 @@ def get_pano_location(pano_id):
     
     # WGS84 to RD conversion 
     point = ogr.Geometry(ogr.wkbPoint)
-    point.AddPoint(geom[0], geom[1])
+    point.AddPoint(max(geom), min(geom))
```

Comment:
This is ugly, but it works because the point has to be in the Netherlands anyway since we're projecting to RD NEW.

Reply:
small thing, but why did you change this in the first place? For me it's clearer with geom[0] and geom[1]

Reply:
The order in which the coordinates are returned seems to have changed, or is inconsistent. Regardless, projecting the coordinates to RD would throw an error. Considering RD is a projection that is only used in the Netherlands, the larger value in the coordinate pair should always precede the smaller value.